frontend/src/app/modules/auth/reset-pass-page/reset-pass-page.component.html

```diff
@@ -51,8 +51,9 @@ <h2>Reset password</h2>
           </button>
         </div>
         <div *ngIf="isDone">
-          If such an email is registered in the database, then the recovery link
-          was sent to the mail
+          If we found an account associated with that username, we've sent
+          password reset instructions to the primary email address on the
+          account.
```

Comment:
You're suggesting to restore a password by email, not by username.

```suggestion
          If we find an account associated with your email,
          we'll send password reset instructions.
```